src/eventDispatchers/mouseEventHandlers/mouseMove.js

```diff
@@ -57,6 +57,12 @@ export default function (evt) {
         data.active = !data.active;
         imageNeedsUpdate = true;
       }
+
+      // TODO: We may want to do _only_ this, and provide a default implementation
```

Comment:
:+1::+1::+1:

I think this makes the most sense. All of this can go into BaseAnnotationTool's virtual mouseMoveCallback, cleans up the readability of this routine too. I think more dispatching of events rather than handling of events here is good.

Reply:
Also a note for the future, we still need to think about what to do for cursors here. I think its the main (only?) use case for mouseMoveCallback.

Between lines 28 and 29, we could filter all tools with an updateCursorPosition(evt) function and call them. This function would most likely just be something along the lines of:

```
  updateCursorPosition (evt) {
    const eventData = evt.detail;


    this._lastImageCoords = eventData.currentPoints.image;
  }
```
Such that the position is known by the class for rerender.

Reply:
Actually! (left my previous comment to demonstrate my thought process).

On mouseMove can't we just put the mousePosition into the mouseEventDispatcher as an export?

```
let mousePosition;
export { mousePosition };
```
In mouseEventHandler.mouseMove:
```
mousePosition = evt.detail.currentPoints.image; // or currentPoints.canvas?
```


Job done, any class that wants a custom cursor can pull this info in at render time. We can now use mouseEventDispatcher.mouseMove to track only hover/unhover.